src/Components/SearchField/SearchField.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import {InputGroup , FormControl} from 'react-bootstrap';
+import { fetchAccountId, fetchMatchHistory } from '../../Modules/APIGateway';
+import GameInfo from '../../Components/GameInfo/GameInfo';
+import './SearchField.css';
+
+function SearchField() {
+
+    const [name, setName] = useState("");
+    const [error, setError] = useState("");
+    const [history,setHistory] = useState([]);
+
+    const handleSubmit = async(evt) => {
+        evt.preventDefault();
```

Comment:
having an async event handler feels bad. It can freeze the user screen until the promise resolves. 
Instead, let's have an effect (https://reactjs.org/docs/hooks-reference.html#useeffect) that listens to state changes and fetches the data accordingly.